Android/app/src/test/java/use_case/dish_list/DishListTest.java

```diff
@@ -98,15 +98,15 @@ public void testDeleteDish() {
         assertFalse(DishList.getAllDishes().containsKey("Milk Tea"));
     }
 
-    /**
```

Comment:
If it is removed, please delete the code. Leaving code in comments will cause warnings